packages/api/src/command/medical/patient/get-adt-subscribers.ts

```diff
@@ -0,0 +1,69 @@
+import { AdtSubscriberData } from "@metriport/core/domain/patient-settings";
+import { capture } from "@metriport/core/util";
+import { out } from "@metriport/core/util/log";
+import { errorToString } from "@metriport/shared";
+import { Sequelize } from "sequelize";
+import { PatientModel } from "../../../models/medical/patient";
+import { PatientSettingsModel } from "../../../models/patient-settings";
+
+export async function getAdtSubscribers(targetStates: string[]): Promise<AdtSubscriberData[]> {
```

Comment:
@RamilGaripov Let's use a better type than string, we have a type that is all possible values of US States and territories, lets use that (though we probably won't be passing in a territory as the value any time soon)